src/bot_functions.py

```diff
@@ -0,0 +1,50 @@
+from pymessenger import Bot
```

Comment:
Also please note that the naming of files is a bit messed up. You have file named "bot_functions.py" which includes class "MessengerBot". The naming of the  file should be adequate to the classes inside. Please rename it to "messenger_bot.py". Please also rename file "messenger_bot.py" to something adequate.